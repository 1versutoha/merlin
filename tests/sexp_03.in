["find", "use", "sexplib"]
["tell", "source", "type t = int with sexp "]
["tell", "source", "let test = sexp_of_t "]
["tell", "end"]
["errors"]
["dump", "env"]
