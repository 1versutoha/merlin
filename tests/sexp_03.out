["return",{"result":true}]
["return",null]
["return",null]
["return",{"line":1,"col":43}]
["return",[]]
["return",[["Line 1, characters 5-13","type t = int"],["Line 1, characters 18-22","val sexp_of_t : t -> Sexplib.Sexp.t"],["Line 1, characters 18-22","val t_of_sexp : Sexplib.Sexp.t -> t"],["Line 1, characters 27-31","val test : t -> Sexplib.Sexp.t"]]]
